CMakeLists.txt

```diff
@@ -297,14 +297,16 @@ set(CPACK_PACKAGE_VERSION_MAJOR ${GUFI_VERSION_MAJOR})
 set(CPACK_PACKAGE_VERSION_MINOR ${GUFI_VERSION_MINOR})
 set(CPACK_PACKAGE_VERSION_PATCH ${GUFI_VERSION_PATCH})
 
-# get the architecture of this machine
-# assumes that we are not cross compiling
-execute_process(COMMAND uname -m OUTPUT_VARIABLE ARCH OUTPUT_STRIP_TRAILING_WHITESPACE)
-
 # Generate RPMs
 find_program(RPMBUILD rpmbuild)
 if (RPMBUILD)
   message(STATUS "Found rpmbuild: ${RPMBUILD_EXECUTABLE}")
+
+  # get the architecture of this machine
+  # assumes that we are not cross compiling
+  find_program(UNAME REQUIRED uname)
```

Comment:
Doesn't seem to be setting uname_executable correctly. Need some checking if(UNAME....). Currently creating RPM w/o arch set.

Reply:
Apparently `find_program` does not have a `REQUIRED` argument. Fixed in 5b0a1034b31416d2b8fc0674a1641d8792668ee3.